packages/server/package.json

```diff
@@ -128,7 +128,7 @@
     "mime-types": "^2.1.16",
     "ms": "^2.0.0",
     "msgpackr": "^1.11.2",
-    "nest-graphql-endpoint": "0.8.1",
+    "nest-graphql-endpoint": "github:ParabolInc/nest-graphql-endpoint",
```

Comment:
+1 Let's do a proper release of that package. Also there is some issue with pnpm and `github:` URLs on CI.